conf/messages.cy

```diff
@@ -469,32 +469,63 @@ pdf.ruling.aboutDecision.text=Rwyf wedi gwneud y penderfyniad hwn o dan adran 24
+pdf.application.footer.solicitors.office.and.legal.services.address=Swyddfa'r Cyfreithiwr a Gwasanaethau Cyfreithiol<br> Solicitor's Office and Legal Services<br> Cyllid a Thollau EF<br> HM Revenue and Customs<br> BX9 1ZT
+
+pdf.ruling.section.appealForIndependentTribunal.title=Os hoffech apelio ar dribiwnlys annibynnol
+pdf.ruling.section.appealForIndependentTribunal.appealImmediately=Os byddai'n well gennych, gallwch apelio'n uniongyrchol ar dribiwnlys annibynnol. Gallwch hefyd symud ymlaen i apêl tribiwnlys unwaith y byddwch wedi cael canlyniad yr adolygiad ac yn anghytuno â'r penderfyniad.
+pdf.ruling.section.appealForIndependentTribunal.courtsAndTribunals=Mae'n rhaid i'r cais gyrraedd Gwasanaeth Llysoedd a Thribiwnlysoedd EF cyn pen 30 diwrnod i ddyddiad y llythyr hwn, neu cyn pen 30 diwrnod i'r dyddiad ar y llythyr sy'n nodi canlyniad yr adolygiad.
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements=Dylai'r cais hwn cynnwys copïau o'r canlynol:
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.1=y llythyr hwn, neu'r llythyr canlyniad yr adolygiad
+pdf.ruling.section.appealForIndependentTribunal.requestRequirements.2=unrhyw asesiad
+pdf.ruling.section.appealForIndependentTribunal.relevantInfo=Os nad ydych yn anfon yr wybodaeth berthnasol, gall y Gwasanaeth Llysoedd a Thribiwnlysoedd EF wrthod eich apêl.
+pdf.ruling.section.appealForIndependentTribunal.tribunalsInfo=I gael rhagor o wybodaeth am dribiwnlysoedd ewch i'r dudalen isod neu ffoniwch 0300 303 5176.
+pdf.ruling.section.appealForIndependentTribunal.page=<a href="www.gov.uk/tribiwnlys-treth" target="_blank">www.gov.uk/tribiwnlys-treth</a>
+
+pdf.ruling.section.appealsAndReviews.title=Rhagor o wybodaeth am apeliadau ac adolygiadau
+pdf.ruling.section.appealsAndReviews.moreInfo=I gael rhagor o wybodaeth am eich hawliau o ran apelio ac adolygiadau, ewch i'r dudalen ganlynol:
+pdf.ruling.section.appealsAndReviews.page=<a href="www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy" target="_blank">www.gov.uk/guidance/how-to-get-a-review-of-an-hmrc-decision.cy</a>
+pdf.ruling.section.appealsAndReviews.copy=Gallwch ofyn am gopi o'r ddogfen HMRC1 drwy ffonio 0300 200 1900.
+
+pdf.ruling.section.communicatingByEmail.title=Gwybodaeth bwysig am ohebu drwy e-bost
+pdf.ruling.section.communicatingByEmail.tellUs=Os hoffech ohebu drwy e-bost, mae'n rhaid i chi roi gwybod i ni eich bod yn deall ac yn derbyn y risgiau o wneud hynny. Manylir ar y rhain yn ein taflen wybodaeth CC/FS72 DSC1, 'Gohebu â CThEF drwy e-bost'. I ddod o hyd i'r daflen wybodaeth hon, ewch i'r dudalen:
+pdf.ruling.section.communicatingByEmail.page=<a href="www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72.cy" target="_blank">www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72.cy</a>
```

Comment:
Please add `class="govuk-link"` within the anchor tag for this link, thanks

Reply:
The href is wrong and redirecting to a not found page. Please use the full url and update the href to `https://www.gov.uk/government/publications/corresponding-with-hmrc-by-email-ccfs72.cy` thanks